js/web/components/HubReleases.js

```diff
@@ -0,0 +1,234 @@
+                  >
+                    <Button
+                      sx={{ cursor: 'pointer' }}
+                      id={release.releasePubkey}
+                      key={release.properties.title}
+                      onClickCapture={(e) =>
+                        handleQueue(
+                          e,
+                          release.releasePubkey,
+                          release.properties.title
+                        )
+                      }
+                    >
+                      <ControlPointIcon
+                        sx={{ color: 'black' }}
+                        key={release.properties.title}
+                        onClickCapture={(e) =>
+                          handleQueue(
+                            e,
+                            release.releasePubkey,
+                            release.properties.title
+                          )
+                        }
+                      />
+                    </Button>
+
+                    <ResponsivePlayButton
+                      onClickCapture={(e) =>
+                        handlePlay(e, release.releasePubkey)
+                      }
+                      id={release.releasePubkey}
+                    >
+                      {isPlaying &&
+                      track.releasePubkey === release.releasePubkey ? (
+                        <PauseCircleOutlineOutlinedIcon
+                          sx={{ color: 'black' }}
+                          onClickCapture={(e) =>
+                            handlePlay(e, release.releasePubkey)
+                          }
+                          id={release.releasePubkey}
+                        />
+                      ) : (
+                        <PlayCircleOutlineOutlinedIcon
+                          sx={{ color: 'black' }}
+                        />
+                      )}
+                    </ResponsivePlayButton>
+                  </StyledTableCell>
+                  <StyledTableCell align="left">
+                    <Box sx={{ width: '50px' }}>
+                      <Image
+                        height={'100%'}
+                        width={'100%'}
+                        layout="responsive"
+                        src={getImageFromCDN(
+                          release.image,
+                          400,
+                          new Date(Date.parse(release.properties.date))
+                        )}
+                        alt={release.name}
+                        priority={true}
+                        loader={loader}
+                      />
+                    </Box>
+                  </StyledTableCell>
+                  <StyledTableCell sx={{ maxWidth: '20vw' }} align="left">
+                    <Box
+                      sx={{
+                        overflow: 'hidden',
+                        textOverflow: 'ellipsis',
+                      }}
+                    >
+                      <Typography noWrap>
```

Comment:
should ink to artist profile

Reply:
needs new indexer to point to publisher address